### PR TITLE
Add flutter to project

### PR DESCRIPTION
**Pull request by seendsouza**

### CODE REVIEW

**Comment by seendsouza** [comment]
rip travis :(

File: lib/src/themes.dart
Comment (jeffrey-dot-li) [inline lib/src/themes.dart:5]: This way of doing fonts and colour schemes is good

File: lib/src/widgets/banner_widget.dart
Comment (jeffrey-dot-li) [inline lib/src/widgets/banner_widget.dart:11]: does flutter automatically make apps responsive for different phone sizes or should we make a note to do that later?

File: pubspec.yaml
Comment (jeffrey-dot-li) [inline pubspec.yaml:3]: Yee part of authors bby

**Review by jeffrey-dot-li — COMMENTED** [review]
What to do about travis? Also in the changes I made, I moved alot of the functionality out of main() and instead into a view part. Is there a way to use the same main() function for both the cli and the flutter app, and is that what we are looking to do?

File: lib/src/widgets/banner_widget.dart
Comment (seendsouza) [reply lib/src/widgets/banner_widget.dart:11]: I haven't tried emulating a tablet, but I think it won't look horrible (like it'll be responsive, but maybe not optimal)

**Comment by seendsouza** [comment]
> What to do about travis? Also in the changes I made, I moved alot of the functionality out of main() and instead into a view part. Is there a way to use the same main() function for both the cli and the flutter app, and is that what we are looking to do?

nope different files (one is considered an executable, and the other is in src, kinda hard to explain why, but trust) 
commands or cli_view if u decide to stray from the commands directory will be for cli views and views will be for flutter views
I'll figure out travis, but until then, we won't merge this into master

**Review by jeffrey-dot-li — APPROVED** [review]
I dont know how ci / the .android & .ios stuff works but it works on my phone so

**Pull request merged**
